internal/stack/serverless.go

```diff
@@ -333,10 +344,10 @@ func (sp *serverlessProvider) TearDown(options Options) error {
 
 	var errs error
 
-	err = sp.destroyLocalAgent()
+	err = sp.destroyLocalService()
 	if err != nil {
-		logger.Errorf("failed to destroy local agent: %v", err)
-		errs = fmt.Errorf("failed to destroy local agent: %w", err)
+		logger.Errorf("failed to destroy local service: %v", err)
+		errs = fmt.Errorf("failed to destroy local service: %w", err)
```

Comment:
Nit. Since there could be more than one service

```suggestion
		logger.Errorf("failed to destroy local services: %v", err)
		errs = fmt.Errorf("failed to destroy local services: %w", err)
```